ethcore/spec/src/spec.rs

```diff
@@ -264,15 +265,29 @@ impl From<ethjson::spec::HardcodedSync> for SpecHardcodedSync {
 	}
 }
 
+impl Serialize for SpecHardcodedSync {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+        where
+            S: Serializer,
+        {
```

Comment:
the indentation seems off here, please make sure that you use tabs instead of spaces.

Reply:
I ran `cargo fmt` before pushing this time, and made sure to use tabs -- please let me know if there is still an issue.

Reply:
Thanks, looks good